SmartDeviceLink/SDLSystemCapability.m

```diff
@@ -82,6 +82,18 @@ - (instancetype)initWithRemoteControlCapability:(SDLRemoteControlCapabilities *)
     return self;
 }
 
+- (instancetype)initWithDisplayCapabilities:(NSArray<SDLDisplayCapability *> *)capabilities {
```

Comment:
This is missing a `.h` initializer. It must have been missed in the widget review.

Reply:
Resolved.